keycloak-theme/.gitignore

```diff
@@ -55,3 +55,4 @@ jspm_packages
 /dist_keycloak
 /build
 /storybook-static
+.env

```

Comment:
This doesn't have to do anything with the keycloak theme's gitignore

Reply:
agree wrong allocation, delete from here